indico/modules/events/operations.py

```diff
@@ -153,11 +153,12 @@ def update_event(event, update_timetable=False, **data):
     _log_event_update(event, changes, visible_person_link_changes=visible_person_link_changes)
 
 
-def clone_event(event, start_dt, cloners, category=None):
+def clone_event(event, n_occurrence, start_dt, cloners, category=None):
     """Clone an event on a given date/time.
 
     Runs all required cloners.
 
+    :param n_occurrence: The number of the occurrence, if this is a "recurring" clone, otherwise `None`
```

Comment:
Let's use `0` in case of non-recurring? Otherwise the `> 0` check will fail in Python 3...

Reply:
Oh, I didn't realize that, good point. Thanks.

Reply:
Ok, done.